src/Identity/ApiAuthorization.IdentityServer/src/Microsoft.AspNetCore.ApiAuthorization.IdentityServer.csproj

```diff
@@ -5,19 +5,29 @@
     <Description>ASP.NET Core API Authorization package powered by Identity Server.</Description>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;apiauth;identity</PackageTags>
+    <HasReferenceAssembly>false</HasReferenceAssembly>
     <IsShippingPackage>true</IsShippingPackage>
+    <HasReferenceAssembly>false</HasReferenceAssembly>
+    <!-- We are a package that depends on the shared framework, this allows us to
+         avoid errors during restore -->
+    <GenerateErrorForMissingTargetingPacks>false</GenerateErrorForMissingTargetingPacks>
   </PropertyGroup>
 
   <ItemGroup>
+    <!-- Remove the reference to Microsoft.AspNetCore.App from IdentityServer4 -->
+    <FrameworkReference Remove="Microsoft.AspNetCore.App" />
     <Reference Include="IdentityServer4.AspNetIdentity" />
     <Reference Include="IdentityServer4.EntityFramework" />
     <Reference Include="IdentityServer4.EntityFramework.Storage" />
     <Reference Include="IdentityServer4" />
     <Reference Include="IdentityServer4.Storage" />
-    <Reference Include="Microsoft.AspNetCore.Authentication.JwtBearer" />
     <Reference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" />
     <Reference Include="Microsoft.AspNetCore.Identity.UI" />
+
+    <!-- These references don't matter as they'll be replaced by the shared framework reference -->
```

Comment:
Not sure this sentence helps much. In any case, it's inconsistent w/ the other projects and you've described the problem already just a few lines earlier.